L1Trigger/L1TTwinMux/python/RPCLinkMap_CondDB_cff.py

```diff
@@ -0,0 +1,29 @@
+import FWCore.ParameterSet.Config as cms
+
+RPCLinkMapSource = cms.ESSource('PoolDBESSource'
+                                , DBParameters = cms.PSet()
+                                , timetype = cms.string('timestamp')
+                                , toGet = cms.VPSet(
+                                    cms.PSet(
+                                        record = cms.string('RPCLBLinkMapRcd')
+                                        , tag = cms.string('RPCLBLinkMap_L1T_v1')
+                                    )
+                                    , cms.PSet(
+                                        record = cms.string('RPCDCCLinkMapRcd')
+                                        , tag = cms.string('RPCDCCLinkMap_L1T_v1')
+                                    )
+                                    , cms.PSet(
+                                        record = cms.string('RPCTwinMuxLinkMapRcd')
+                                        , tag = cms.string('RPCTwinMuxLinkMap_L1T_v1')
+                                    )
+                                    , cms.PSet(
+                                        record = cms.string('RPCCPPFLinkMapRcd')
+                                        , tag = cms.string('RPCCPPFLinkMap_L1T_v1')
+                                    )
+                                    , cms.PSet(
+                                        record = cms.string('RPCOMTFLinkMapRcd')
+                                        , tag = cms.string('RPCOMTFLinkMap_L1T_v1')
+                                    )
```

Comment:
and here

Reply:
Hi @arunhep 
The file L1Trigger/L1TTwinMux/python/RPCLinkMap_CondDB_cff.py is not used and its modules are not imported anywhere in this PR, so having this config file does not hurt anything.  
All the RPC*LinkMaps are accessed from GTs, as it should be.